libs/core/functional/include/hpx/functional/mem_fn.hpp

```diff
@@ -39,22 +40,77 @@ namespace hpx { namespace util {
         };
     }    // namespace detail
 
+    /// \brief Function template \c hpx::mem_fn generates wrapper objects for pointers
+    ///        to members, which can store, copy, and invoke a pointer to member.
+    ///        Both references and pointers (including smart pointers) to an object
+    ///        can be used when invoking a \c hpx::mem_fn.
+    ///
+    /// \param pm pointer to member that will be wrapped
+    ///
+    /// \return a call wrapper of unspecified type with the following member:
```

Comment:
@hkaiser can you please check if the return type for this is correct? I used the corresponding cpp standard as reference: https://en.cppreference.com/w/cpp/utility/functional/mem_fn

Reply:
This is fine, thanks!